### DISCUSSION TITLE
wire:navigate prevents wire:loading directives to work on that element

### DISCUSSION THREAD

**Discussion by angus-mcritchie** [livewire/livewire#9057] [7. Bugs (archived)] [answered]
### Livewire version

3.5.15

### Laravel version

10.47.0

### Which PHP version are you using?

PHP 8.1

### Steps To Reproduce

1. Add wire:navigate and wire:loading on an element
2. Trigger an action that makes the livewire component to load
3. See that the wire:loading directive has no effect

### Repository or Wirebox URL

https://wirebox.app/b/x9nv0

### Additional information

Is this the expected behaviour? I couldn't find anything in the docs in both the [wire:loading](https://livewire.laravel.com/docs/wire-loading) and [wire:navigate](https://livewire.laravel.com/docs/navigate) sections.

### Please confirm (incomplete submissions will not be addressed)

- [X] I have provided easy and step-by-step instructions to reproduce the bug.
- [X] I have provided code samples as text and NOT images.
- [X] I understand my bug report will be removed if I haven't met the criteria above.

**Comment by danie-ramdhani** [comment]
add `wire:target.except` https://livewire.laravel.com/docs/wire-loading#excluding-specific-loading-targets

**Reply by angus-mcritchie** [reply]
Thanks, @danie-ramdhani, that does work. Do you know if this is the expected behaviour?

If so, we should add this to the docs, as it's not obvious.

Otherwise, maybe we should make this just work without adding `wire:target.except`?

**Reply by danie-ramdhani** [reply]
wdym its should add to the docs? it's already documentated. https://livewire.laravel.com/docs/wire-loading#excluding-specific-loading-targets. and it is expected behavior. why not create link component with that `wire:target.except` as default behavior?

**Reply by angus-mcritchie** [reply]
@danie-ramdhani I mean, it's not obvious that adding a `wire:navigate` will change the default behaviour of `wire:loading`.

I have updated the [wirebox](https://wirebox.app/b/x9nv0) to illustrate better what I mean.

**Comment by joshhanley** [comment, marked as answer]
@angus-mcritchie thanks for reporting! I've submitted a PR #9378 with a fix.
